omaha/base/const_addresses.h

```diff
@@ -58,26 +58,26 @@ const TCHAR* const kGoopdateServer = _T("tools.") COMPANY_DOMAIN;
 // The channel for update checks is secured by using CUP to sign the messages.
 // It does not depend solely on https security in any case.
 const TCHAR* const kUrlUpdateCheck =
-    _T("https://update.") COMPANY_DOMAIN_BASE _T("apis.com/service/update2");
+    _T("http://localhost:9090/service/update2");
```

Comment:
oooops, I think that we need have URL base on env here. so I drop the approve

Reply:
new URL:

- https://update-dev.cthesigns.co.uk/service/update2
- https://update.cthesigns.co.uk/service/update2

should be picked base on build branch

Reply:
done, so far i changed it to dev server, my plan is to include that address as tag for tagged metainstaller so that we can quickly generate exe that points to specific server